src/test/java/kitchenpos/bo/MenuBoTest.java

```diff
@@ -0,0 +1,286 @@
+package kitchenpos.bo;
+
+import kitchenpos.builder.MenuBuilder;
+import kitchenpos.builder.MenuGroupBuilder;
+import kitchenpos.builder.MenuProductBuilder;
+import kitchenpos.builder.ProductBuilder;
+import kitchenpos.dao.MenuDao;
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.dao.MenuProductDao;
+import kitchenpos.dao.ProductDao;
+import kitchenpos.model.Menu;
+import kitchenpos.model.MenuGroup;
+
+import kitchenpos.model.MenuProduct;
+import kitchenpos.model.Product;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.math.BigDecimal;
+import java.util.*;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(MockitoExtension.class)
+class MenuBoTest {
+    @Mock
+    private MenuDao menuDao;
+
+    @Mock
+    private MenuGroupDao menuGroupDao;
+
+    @Mock
+    private MenuProductDao menuProductDao;
+
+    @Mock
+    private ProductDao productDao;
+
+    @InjectMocks
+    private MenuBo menuBo;
+
+    private MenuGroupBuilder menuGroupBuilder;
+    private ProductBuilder productBuilder;
+    private MenuBuilder menuBuilder;
+    private MenuProductBuilder menuProductBuilder;
+
+    @BeforeEach
+    void setUp() {
+        productBuilder = new ProductBuilder();
+        menuGroupBuilder = new MenuGroupBuilder();
+        menuBuilder = new MenuBuilder();
+        menuProductBuilder = new MenuProductBuilder();
+    }
+
+    @Test
+    @DisplayName("메뉴를 새로 생성활 수 있다")
+    void createMenu() {
+        Product product = productBuilder
+                .id(1L)
+                .name("후라이드")
+                .price(BigDecimal.valueOf(16000))
+                .build();
+
+        MenuGroup menuGroup = menuGroupBuilder
+                .id(1L)
+                .name("")
+                .build();
+
+        Menu newMenu = menuBuilder
+                .id(1L)
+                .name("후라이드치킨")
+                .menuGroupId(menuGroup.getId())
+                .price(BigDecimal.valueOf(16000))
+                .menuProducts(asList(menuProductBuilder
+                        .seq(1L)
+                        .menuId(1L)
+                        .productId(product.getId())
+                        .quantity(1)
+                        .build()
+                ))
+                .build();
+
+        given(menuGroupDao.existsById(anyLong()))
```

Comment:
`anyLong()`이 아닌 위에 정의한 MenuGroup의 아이디를 넣어주면 좀 더 명확한 테스트가 될 수 있지 않을까요?

 ```suggestion
        given(menuGroupDao.existsById(menuGroup.getId()));
```